app/controllers/agenda/agendaitems/agendaitem.js

```diff
@@ -19,11 +19,11 @@ export default class AgendaAgendaitemsAgendaitemController extends Controller {
 
   get canShowDecisionTab() {
     return this.currentSession.may('manage-decisions')
-      || (this.meeting.isFinal && this.hasDecision);
+      || (this.meeting.agenda && this.hasDecision);
```

Comment:
Since `this.meeting.agenda` returns a promise, `this.meeting.agenda` will always be thruthy, even if the promise resolves to `null`. You should check whether it resolves to an actual agenda, for example by checking the existance of `agenda.id`. In practice it might be that the promise immediately resolves if the data has already been loaded, but that's not something you may rely on.

Since `this.meeting.agenda` is a promise, you will need to use `get()` to retrieve the `id` property, like `this.meeting.agenda.get('id')`.

To stay in line with the remainder of this file it's probably best to define a `get isFinalAgenda() { ... }` similar to `get hasDecision()` and `get hasNewsItem()`.